cast-dssspell.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$(seth chain --rpc-url="$ETH_RPC_URL")" == "goerli" ]] || { echo "Please set a Goerli ETH_RPC_URL"; exit 1; }
+
+CHANGELOG=0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F
+MCD_ADM=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_ADM")")")
+MCD_GOV=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_GOV")")")
+MCD_IOU=$(seth call "$CHANGELOG" 'getAddress(bytes32)(address)' "$(seth --to-bytes32 "$(seth --from-ascii "MCD_IOU")")")
+MKR=$(seth --to-wei 10001 ether)
+
+if [[ -z "$1" ]]; then
+    echo "Please specify the Goerli Spell Address"
+else
+    seth send "$MCD_GOV" 'approve(address,uint256)' "$MCD_ADM" "$MKR"
+    seth send "$MCD_ADM" 'lock(uint256)' "$MKR"
```

Comment:
Can we add a balance check of the `ETH_FROM` address to ensure that there's enough MKR available in the wallet? If not, short-circuit and provide an error.

Reply:
done in b001a0d

Reply:
Should we keep the MKR amount to lock/free hard coded or pass it to the script as parameter?

Reply:
attempted to generalize the script in 5fe7ad8.